Firmware/PlatformIO/main_mcu/src/bms.h

```diff
@@ -16,6 +18,11 @@ const float kCellOvervoltage = 4.2;
 const float kInternalResistance = 0.016;
 const int kNumCellsParallel = 4;
 
+// CAN Bus Numbers
+const int kHPBusNumber = 1;
+const int kVBBusNumber = 2;
+const int kLPBusNumber = 3;
```

Comment:
These should be private member variables (you probably have to make them static or the compiler won't like how they're used)